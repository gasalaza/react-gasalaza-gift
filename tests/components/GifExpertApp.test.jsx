import { render, screen } from "@testing-library/react";
import { GifExpertApp } from "../../src/GifExpertApp";

describe('prueba en GifExpertApp', () => { 
    
    test('debe de hacer match con el snapshot', () => { 
        const { container } = render( <GifExpertApp /> );
        expect( container ).toMatchSnapshot();
     });

     test('debe de mostrar el componente <AddCategory />', () => { 
        render( <GifExpertApp /> );
        expect( screen.getByText('GifExpertApp') ).toBeTruthy();
      });

      test('debe de mostrar una lista de categorias', () => { 
        render( <GifExpertApp /> );
        expect( screen.getByText('Punisher') ).toBeTruthy();
      }); 

});