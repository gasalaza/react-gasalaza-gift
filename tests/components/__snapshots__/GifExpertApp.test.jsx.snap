// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prueba en GifExpertApp debe de hacer match con el snapshot 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="Buscar Gif"
      type="text"
      value=""
    />
  </form>
  <h3>
    Punisher
  </h3>
  <h2>
    Loading...
  </h2>
  <div
    class="card-grid"
  />
</div>
`;
